tests/specs/v2/rumXhrStatusTracking.js

```diff
@@ -42,75 +52,71 @@ describe("RUM status code tracking", function() {
       var pairStatus = expectedPairs[i].statusCode;
       var pairType = expectedPairs[i].type;
 
-      expect(payloadUrl).toBe(pairUrl, "failed for type: " + pairType);
-      expect(payloadStatus).toBe(pairStatus, "failed for type: " + pairType);
-      expect(payloadDataType).toBe(payloadDataType, "XHR data type missing for: " + pairType);
+      expect(payloadUrl === pairUrl).toBeTrue("failed for type: " + pairType);
+      expect(payloadStatus === pairStatus).toBeTrue( "failed for type: " + pairType);
+      expect(payloadDataType === payloadDataType).toBeTrue( "XHR data type missing for: " + pairType);
     }
   }
 
 
-  it("attaches the status codes to xhr calls for XmlHttpRequest", function () {
-    browser.url('http://localhost:4567/fixtures/v2/rumXhrStatusCodes.html');
+  it("attaches the status codes to xhr calls for XmlHttpRequest", async function () {
+    await browser.url('http://localhost:4567/fixtures/v2/rumXhrStatusCodes.html');
 
-    browser.pause(35000);
+    await browser.pause(40000);
 
-    checkStatusCodes();
+    await checkStatusCodes();
   });
 
-  it("attaches status codes to requests for fetch requests", function() {
-    browser.url('http://localhost:4567/fixtures/v2/rumFetchStatusCodes.html');
+  it("attaches status codes to requests for fetch requests", async function() {
+    await browser.url('http://localhost:4567/fixtures/v2/rumFetchStatusCodes.html');
 
-    browser.pause(1000);
+    await browser.pause(40000);
```

Comment:
1s to 40s here?

Reply:
Yes. for some tests it needed to be 40s, to complete reliably

Reply:
Made this better that it will only pause as long as it needs to (up to 40 seconds)